### ISSUE TITLE
AWS SNS Trigger - Error: subscriptions is not iterable

### ISSUE THREAD

**Issue by roemhildtg** [n8n-io/n8n#1058]
**Describe the bug**

When I enable the SNS trigger in a workflow, and I try to save the workflow, I get an error:

```
subscriptions is not iterable

```

**To Reproduce**
Use workflow for example:

```json
{
  "nodes": [
    {
      "parameters": {
        "topic": "arn:aws:sns:us-east-1:xxxxxx"
      },
      "name": "AWS-SNS-Trigger",
      "type": "n8n-nodes-base.awsSnsTrigger",
      "typeVersion": 1,
      "position": [
        390,
        300
      ],
      "webhookId": "xxxxxxx",
      "credentials": {
        "aws": "xxxxxx"
      }
    }
  ],
  "connections": {}
}
```

**Expected behavior**

I think there's some error in the parsing logic of the aws subscription response.

**Environment (please complete the following information):**
 - OS: Docker (Ubuntu trusty)

![image](https://user-images.githubusercontent.com/5471079/96018987-88f62e00-0e11-11eb-8220-5f9b778ef5db.png)

Additional:

The AWS User I'm using has the following policy(s):

```json
{
    "Version": "2012-10-17",
    "Statement": [
        {
            "Sid": "VisualEditor0",
            "Effect": "Allow",
            "Action": [
                "sns:CreatePlatformApplication",
                "sns:SetSMSAttributes",
                "sns:ListTopics",
                "sns:GetPlatformApplicationAttributes",
                "sns:CreatePlatformEndpoint",
                "sns:Unsubscribe",
                "sns:GetSubscriptionAttributes",
                "sns:ListSubscriptions",
                "sns:CheckIfPhoneNumberIsOptedOut",
                "sns:OptInPhoneNumber",
                "sns:DeleteEndpoint",
                "sns:SetEndpointAttributes",
                "sns:ListPhoneNumbersOptedOut",
                "sns:ListEndpointsByPlatformApplication",
                "sns:GetEndpointAttributes",
                "sns:SetSubscriptionAttributes",
                "sns:DeletePlatformApplication",
                "sns:SetPlatformApplicationAttributes",
                "sns:GetSMSAttributes",
                "sns:ListPlatformApplications"
            ],
            "Resource": "*"
        },
        {
            "Sid": "VisualEditor1",
            "Effect": "Allow",
            "Action": "sns:*",
            "Resource": "arn:aws:sns:us-east-1:xxxx:*"
        },
        {
            "Sid": "VisualEditor2",
            "Effect": "Allow",
            "Action": [
                "sns:ListTagsForResource",
                "sns:ListSubscriptionsByTopic",
                "sns:GetTopicAttributes",
                "sns:ListTopics",
                "sns:GetPlatformApplicationAttributes",
                "sns:GetSubscriptionAttributes",
                "sns:ListSubscriptions",
                "sns:CheckIfPhoneNumberIsOptedOut",
                "sns:ListPhoneNumbersOptedOut",
                "sns:ListEndpointsByPlatformApplication",
                "sns:GetEndpointAttributes",
                "sns:ListPlatformApplications",
                "sns:GetSMSAttributes"
            ],
            "Resource": "*"
        }
    ]
}
```

**Comment by RicardoE105** [comment]
@roemhildtg was able to reproduce it. It's quite weird as it works with testing but not when you activate it. Investigating will keep you posted. Thanks.

**Comment by RicardoE105** [comment]
@roemhildtg got fixed. We will let you know when is released.

**Comment by roemhildtg** [comment]
Thank you! That was super fast!

**Comment by janober** [comment]
Thanks a lot @RicardoE105! Just got merged.

The new version with the fix will get released tomorrow with a new version. I will update here as soon as it is available.

**Comment by janober** [comment]
Fix got released with n8n@0.88.0
